.travis.yml

```diff
@@ -7,6 +7,10 @@ install:
   - pip install coverage
 services:
   - postgresql
+  
+addons:
+  postgresql: "9.4"
```

Comment:
@PAbishai was this necessary?

Reply:
I was getting this error from Travis "ProgrammingError: syntax error at or near "WITH ORDINALITY" and the solution was to enable use of Postgres 9.4 or higher

Reply:
Okay